lego/apps/users/tests/test_models.py

```diff
@@ -109,6 +109,26 @@ def test_set_profile_picture(self):
         self.user.save()
         self.assertEqual(self.user.profile_picture, "abakus.png")
 
+    def test_remove__male_profile_picture(self):
```

Comment:
I don't think you need these tests either, since you're not really adding any functionality, but only changing a field to nullable